docs/guides/debug-qiskit-runtime-jobs.ipynb

```diff
@@ -0,0 +1,517 @@
+{
```

Comment:
<p>Perhaps link to <a href="https://qiskit.github.io/documentation/pr-2223/guides/simulate-with-qiskit-aer" target="_blank">https://qiskit.github.io/documentation/pr-2223/guides/simulate-with-qiskit-aer</a> so people who are new to Qiskit Aer noise model know where to find more information.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>